api/api.go

```diff
@@ -343,9 +343,10 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 		}
 
 		apiAlert := &dispatch.APIAlert{
-			Alert:     &a.Alert,
-			Status:    status,
-			Receivers: receivers,
+			Alert:       &a.Alert,
+			Status:      status,
+			Receivers:   receivers,
+			Fingerprint: fmt.Sprintf("%x", a.Fingerprint()),
```

Comment:
@prymitive When I curl Alertmanager it does not expose the Fingerprint base16 encoded. I looked through the `common` package and found the  [`String`](https://github.com/prometheus/alertmanager/blob/master/vendor/github.com/prometheus/common/model/fingerprinting.go#L40) function. Would you mind changing it again to `Fingerprint: a.Fingerprint().String(),` then we are consistent across projects. I know there are plans to get rid of the `commons` repo in the future, but as we are using `Fingerprint` from the repo anyways, we might as well use `Fingerprint.String()`. Let me know if this sounds reasonable to you.

//cc @fabxc